README.md

```diff
@@ -30,13 +30,19 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document object model- is an interface for html that allows the document to be manipulated
```

Comment:
A little more elaboration would be nice. Think about answering this for an interview. How would you answer it?